frontend/packages/console-app/src/plugin.tsx

```diff
@@ -81,7 +82,7 @@ const plugin: Plugin<ConsumedExtensions> = [
     type: 'Perspective',
     properties: {
       id: 'admin',
-      name: 'Administrator',
+      name: i18next.t('console-app~Administrator'),
```

Comment:
@rebeccaalpert Sorry, I know this is merged, but we shouldn't be using `i18next.t` here. It should be...

```
      // t('console-app~Administrator')
      name: '%console-app~Administrator%',
```

Reply:
Disregard, I see this has since been fixed.